.github/pull_request_template.md

```diff
@@ -4,7 +4,7 @@
 
 Please include the type of fix and description of your change according to the (Conventional Commits specification)[https://www.conventionalcommits.org/en/v1.0.0/]. 
 
-Fixes # (issue)
+fixes # (issue)
```

Comment:
Any reason for this change? Or is this an accident?

Reply:
I think because the PR was originally based off main, its trying to merge this commit (which is already merged in main) into the develop branch as well.

Reply:
Yeah, I made this change because Github wasn't auto-closing issues when it was "Fixes"

Reply:
Oh, gotcha. Thanks